inlong-sort-standalone/sort-standalone-source/pom.xml

```diff
@@ -42,5 +42,39 @@
             <artifactId>sort-standalone-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>sort-sdk</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+            <version>2.0.2</version>
```

Comment:
abstract version to properties

Reply:
It seems to be a coincidence that some dependecies have the same version.  In fact, not like dependencies of inlong, they are not associated with each other.  Hence, I don't think it's necessary to abstract version "2.0.2" to properties.